microsoftgraph/microsoft-graph-explorer-v2
# Feature: Copy response headers

## Overview

Enables copying the response header.
Fixes #308 

### Demo

![image](https://user-images.githubusercontent.com/58787602/72419485-fc3d1780-378d-11ea-9cbc-a2ea7fa7d450.png)

## Testing Instructions

* Send a request
* Open the response tab
* Notice the copy button
